requirements.txt

```diff
@@ -1,6 +1,7 @@
-pytest==2.7.3
+BeautifulSoup==3.2.1
+browserid
```

Comment:
This was renamed to `bidpom`. I'm surprised we didn't update in all our requirements files. This can be a separate commit, or even a separate pull.